src/swap.ts

```diff
@@ -322,6 +337,123 @@ export class Swap {
     }
   }
 
+  /**
+   * Get the Alpha deploy transaction.
+   *
+   * @returns null if cnd hasn't seen a deploy transaction, otherwise, {@link Transaction} if supported or the transaction id as string.
+   */
+  public async getAlphaDeployTransaction(): Promise<
```

Comment:
At some point we will have to think how we are going to `scale` the `Swap` class with more protocols being added and all of them having different actions ..

Reply:
1. I hesitated between fully-qualified function names or passing a `Alpha`/`Beta` and `Deploy`/`Fund`/`Redeem`/`Refund`.
2. I would like to see the actions going away once @yosriady is done with the first PoC and is keen to give an idea of the ideal interface.